VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/hearing-rules/hearing-rules.component.ts

```diff
@@ -12,6 +13,7 @@ import { Logger } from 'src/app/services/logging/logger-base';
 })
 export class HearingRulesComponent extends ParticipantStatusBaseDirective implements OnInit {
     conferenceId: string;
+    hearingVenueIsScottish$: Observable<boolean>;
```

Comment:
Can this be removed now?

Reply:
yea, also spotted few other places I missed out